pkgs/applications/misc/nhentai/default.nix

```diff
@@ -29,7 +29,7 @@ python3Packages.buildPythonApplication rec {
 
   meta = {
     homepage = "https://github.com/RicterZ/nhentai";
-    description = "nHentai is a CLI tool for downloading doujinshi from <http://nhentai.net>";
+    description = "nHentai is a CLI tool for downloading doujinshi";
```

Comment:
```suggestion
    description = "CLI tool for downloading doujinshi from adult site(s)";
```

"from adult site(s)" as you suggested seems more succinct and gets the point across.
Removing the reference to the package name as per https://nixos.org/manual/nixpkgs/unstable/#var-meta-description:
> Don’t include a period at the end. Don’t include newline characters. Capitalise the first character. For brevity, don’t repeat the name of package — just describe what it does.

As for the by-name check, it's fine to move it, in a separate commit.

Reply:
Also I should point out - the CI check is failing on `pkgs.openobserve`, not this package.

Reply:
Thank you, this is a better description.